cmd/plugin_manager.go

```diff
@@ -117,3 +117,26 @@ func createPluginManagerLog() hclog.Logger {
 	log.SetFlags(0)
 	return logger
 }
+
+func cleanupOldTmpDirs() {
```

Comment:
Move this function into the [plugin](https://github.com/turbot/steampipe/tree/main/pkg/plugin) package.